bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,44 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    class << self
+      attr_accessor :machine_id
+    end
+
+    def report_runtime(addtl_dimensions, value)
+      log_metric(:Runtime, addtl_dimensions, value)
+    end
+
+    def get_machine_id
+      begin
+        metadata_endpoint = 'http://169.254.169.254/latest/meta-data/'
+        return Net::HTTP.get(URI.parse(metadata_endpoint + 'instance-id'))
+      rescue
+        return "local_machine"
+      end
+    end
+
+    def log_metric(metric_name, addtl_dimensions, value)
+      machine_id = I18n::Metrics.machine_id ||= get_machine_id
```

Comment:
1. There is no need to add `return` at the end of a method
2. https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-string-quotes
```suggestion
    def self.machine_id
      @machine_id ||= begin
        metadata_endpoint = 'http://169.254.169.254/latest/meta-data/'
        Net::HTTP.get(URI.parse(metadata_endpoint + 'instance-id'))
      rescue
        'local_machine'
      end
    end

    def self.log_metric(metric_name, addtl_dimensions, value)
```

Reply:
yep fair! I find it more readable with a return statement but happy to follow Ruby convention as well